examples/MLOps.NET.Examples/MLOps.NET.Clustering/Program.cs

```diff
@@ -0,0 +1,62 @@
+﻿using Microsoft.ML;
+using Microsoft.ML.Trainers;
+using MLOps.NET.Clustering.Entities;
+using MLOps.NET.SQLite;
+using System;
+
+namespace MLOps.NET.Clustering
+{
+    class Program
+    {
+        static async System.Threading.Tasks.Task Main(string[] args)
+        {
+            // MLOps: Create experiment and run
+            var mlOpsContext = new MLOpsBuilder()
+                .UseSQLite()
+                .Build();
+
+            Console.WriteLine("Creating an MLOps Run");
+            var runId = await mlOpsContext.LifeCycle.CreateRunAsync("Iris Predictor");
+            Console.WriteLine($"Run created with Id {runId}");
+
+            var mlContext = new MLContext(seed: 1);
+
+            Console.WriteLine("Loading the data");
+            var data = mlContext.Data.LoadFromTextFile<ModelInput>("Data/iris-dataset.csv", hasHeader: true, separatorChar: ',');
+            var testTrainTest = mlContext.Data.TrainTestSplit(data);
+            var features = new[] { nameof(ModelInput.SepalLengthCm), nameof(ModelInput.SepalWidthCm), nameof(ModelInput.PetalLengthCm), nameof(ModelInput.PetalWidthCm) };
+
+            Console.WriteLine("Creating a data processing pipeline");
+            var dataProcessingPipeline = mlContext.Transforms.Concatenate("Features", features)                
+                .Append(mlContext.Transforms.NormalizeMinMax("Features"));
+
+            Console.WriteLine("Training the model, please stand-by...");
+            var trainer = mlContext.Clustering.Trainers.KMeans(featureColumnName: "Features", numberOfClusters: 3);
+            var trainingPipeline = dataProcessingPipeline
+                .Append(trainer);
+
+            var trainedModel = trainingPipeline.Fit(testTrainTest.TrainSet);
+
+            await mlOpsContext.Training.LogHyperParametersAsync<KMeansTrainer>(runId, trainer);
+
+            //MLOps: Training time
+            await mlOpsContext.LifeCycle.SetTrainingTimeAsync(runId);
+            Console.WriteLine($"Training time:{mlOpsContext.LifeCycle.GetRun(runId).TrainingTime}");
+
+            Console.WriteLine("Evaluating the model");
+            var predictions = trainedModel.Transform(testTrainTest.TestSet);
+            var metrics = mlContext.Clustering.Evaluate(predictions, "Label", "Score", "Features");
```

Comment:
I think `Label`, `Score`, and `Features` are the default values, so you probably don't need to override this method, e.g. `mlContext.Clustering.Evaluate(predictions);` should be enough

Reply:
This was done to ensure the NormalizedMutualInformation always return a value and not NaN (#182 ). I wasn't sure if this would be merged before the handle. So, I played it safe. I'll update now.